horizons/world/units/ship.py

```diff
@@ -154,9 +154,9 @@ def _update_buoy(self, remove_only=False):
 
 		ship_id = self.worldid
 		session = self.session # this has to happen here,
+
 		# cause a reference to self in a temporary function is implemented
 		# as a hard reference, which causes a memory leak
-
 		def tmp():
```

Comment:
Sorry, no. The comment belongs to the `session = ` line, but the `session = ` line belongs to the function. So imho there shouldn't be any newline.